server/src/main/java/org/elasticsearch/cluster/routing/RoutingNode.java

```diff
@@ -23,11 +23,7 @@
 import org.elasticsearch.common.Nullable;
 import org.elasticsearch.index.shard.ShardId;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.List;
+import java.util.*;
```

Comment:
Wildcard imports are forbidden and will fail the build. You can run `./gradlew precommit` to check this.